cmd/tarmak/cmd/cluster_logs.go

```diff
@@ -0,0 +1,42 @@
+// Copyright Jetstack Ltd. See LICENSE for details.
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/jetstack/tarmak/pkg/tarmak"
+	"github.com/jetstack/tarmak/pkg/tarmak/logs"
+)
+
+// clusterLogsCmd handles `tarmak clusters logs`
+var clusterLogsCmd = &cobra.Command{
+	Use: "logs [target groups]",
+	Long: fmt.Sprintf(
+		"Gather logs from a list of instances or target groups %s",
+		logs.TargetGroups,
+	),
+	Short: "Gather logs from a list of instances or target groups",
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
```

Comment:
Some better validation of targets would be great.

Reply:
I've done more validation further down. I don't want to call ListHosts() here again because it's so expensive